src/govuk/components/pagination/_index.scss

```diff
@@ -0,0 +1,204 @@
+@include govuk-exports("govuk/component/pagination") {
+  // Flexbox enhancement for small screen visual design
+  // Falls back to a float: left layout on non-flex browsers
+  .govuk-pagination__list {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    // Use the before and after pseudo elements to create "line breaks" in the flex elements
+    // between the prev and next buttons and the numbers on mobile
+    &:before,
+    &:after {
+      content: "";
+      display: block;
+      flex-basis: 100%;
+    }
+
+    &:before {
+      order: 1;
+    }
+
+    &:after {
+      order: 3;
+    }
+
+    @include govuk-media-query($from: tablet) {
+      justify-content: flex-start;
+
+      &:before,
+      &:after {
+        flex-basis: 0;
+      }
+    }
+  }
+
+  .govuk-pagination__item {
+    @include govuk-font(19);
+    display: none; // Hide items on small screens except the prev/next items, non-link items and the first and last items
+    position: relative;
+    margin-bottom: govuk-spacing(4);
+    float: left; // Float is ignored if flex is active
+    order: 2;
+
+    // Hide items on small screens
+    @include govuk-media-query($from: tablet) {
+      display: block;
+    }
+
+    &:hover {
+      background-color: govuk-colour("light-grey", $legacy: "grey-4");
+    }
+  }
+
+  .govuk-pagination__icon {
+    // Set size using rems to make the icon scale with text if user resizes text in their browser
+    width: govuk-px-to-rem(15px);
+    height: govuk-px-to-rem(13px);
+    margin-top: govuk-spacing(1);
+    fill: $govuk-secondary-text-colour;
+  }
+
+  .govuk-pagination__icon--prev {
+    margin-right: govuk-spacing(3);
+  }
+
+  .govuk-pagination__icon--next {
+    margin-left: govuk-spacing(3);
+  }
+
+  // Only show previous, next, first, last and non-link items on mobile
+  .govuk-pagination__item--current,
+  .govuk-pagination__item--ellipses,
+  .govuk-pagination__item--prev,
+  .govuk-pagination__item--next,
+  .govuk-pagination__item:nth-child(2),
+  .govuk-pagination__item:nth-last-child(2) {
+    display: block;
+  }
+
+  .govuk-pagination__item,
+  .govuk-pagination__item--current,
+  .govuk-pagination__item--ellipses {
+    box-sizing: border-box;
+    min-width: govuk-spacing(8);
```

Comment:
Are these _really_ tied to the spacing scale? If not, might make more sense to just use `50px`.